BigData_ExamineAetrexDB.R

```diff
@@ -0,0 +1,50 @@
+rm(list=ls())
+library(tidyverse)
+library(readxl)
+library(lme4)
+library(patchwork)
+library(effsize)
+
+#-------------------------------------------------------------------------------
+# The purpose of this code is to understand the differences between male and 
+# female foot anthroprometrics
+#-------------------------------------------------------------------------------
+# Loading in the data frame and organizing left and right sides into date frames
+footAnth <- read.csv('C:/Users/eric.honert/Boa Technology Inc/PFL Team - General/BigData/FootScan Data/March23_export.csv')
+colnames(footAnth)[which(names(footAnth) == 'Gender')] <- 'Sex'
+
+footAnth$�..Loc
+
+NAdat <- footAnth %>%
```

Comment:
In the past I have had issues with the " ï " character. Not sure if that is what is causing my error below.

Getting this error: 

Error in `filter()`:
! Problem while computing `..1 = ... | ï..Location == "United States"`.
Caused by error in `mask$eval_all_filter()`:
! object 'ï..Location' not found